Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM debian:stable-slim
+
+ENV BEASTPORT=30005 \
+    S6_BEHAVIOUR_IF_STAGE2_FAILS=2
+
+RUN set -x && \
+   apt-get update && \
+    apt-get install --no-install-recommends -y \
```

Comment:
We can probably reduce what we install (and keep the image size down. 

Looking at `rootfs/usr/bin/adsbhub.sh`, we'll need:

* ca-certificates
* wget (can likely swap curl out for this)
* net-tools (for netstat)
* socat

And temporarily (during container build), we'll need:

* gnupg (to verify s6-overlay)
* file (so the s6-overlay installer script can determine the correct architecture)

Have a look at how I've started managing this here: <https://github.com/mikenye/docker-readsb-protobuf/blob/54edf941e71bcb9aaaca63cbe35d0bf9d4d69b7b/Dockerfile#L65>

The `KEPT_PACKAGES` packages are installed. The `TEMP_PACKAGES` packages are removed curing the clean-up at the end of the container build: <https://github.com/mikenye/docker-readsb-protobuf/blob/54edf941e71bcb9aaaca63cbe35d0bf9d4d69b7b/Dockerfile#L258>